MarvellOS.py

```diff
@@ -5,9 +5,19 @@
 from threading import Thread
 from tkinter import *
 import base64
+import glob
+import numpy as np
+import PIL
+from PIL import Image,ImageTk
 
 WINDOW_DIM = "325x400"
 
+#Tkinter Related
+import tkinter.filedialog as tkfd
+from tkinter import *
```

Comment:
- This import is already made at the top.
- Please add all the imports before the `WINDOW_DIM` line (imports are supposed to be at the very top of the file).